company-profiles/swimlane.md

```diff
@@ -0,0 +1,29 @@
+# Swimlane
+
+## Company blurb
+
+Swimlane is a rapidly growing, innovative startup in the Denver metro area with a passionate team focused on improving the lives of security practitioners through security automation and orchestration. We’re looking for dedicated, ambitious problem solvers who thrive in a fast-paced work environment to join our team. At Swimlane, you’ll have the opportunity to be a pioneer working with a cutting-edge technology stack to create and distribute products that empower SecOps teams to automate time consuming and repetitive tasks, letting them focus on the advanced security activities that get the most use out of their training.
+
+## Company size
+
+150-200
+
+## Remote status
+
+We’re headquartered in the Denver, CO metro area, but our employees span 23 states and 6 countries.
```

Comment:
"6 countries" doesn't match with the region being USA - does the company hire only in the US?

Reply:
@dougaitken We're currently looking just in the US but have hired international previously.